src/coreclr/jit/hwintrinsicxarch.cpp

```diff
@@ -2361,7 +2362,7 @@ GenTree* Compiler::impSpecialIntrinsic(NamedIntrinsic        intrinsic,
             }
             else if (simdSize == 64)
             {
-                if (varTypeIsByte(simdBaseType))
+                if (varTypeIsByte(simdBaseType) && !compExactlyDependsOn(InstructionSet_AVX512VBMI))
```

Comment:
This and the other two should be `compOpportunisticallyDependsOn` since the fallback is the same behavior, just slower.

Reply:
Done

Reply:
This one is still `exactly` and should be `opportunistically`

Reply:
Missed a couple there- changed now